config/dev/dgutman-dsa-server-preprod-ec2.yaml

```diff
@@ -0,0 +1,27 @@
+# This stack is based on `config/prod/dgutman-htan-new2-ec2.yaml`
+# in https://github.com/Sage-Bionetworks/sandbox-provisioner/
+template_path: "remote/managed-ec2-linux-v2.j2"
+stack_name: "dgutman-dsa-server-preprod-ec2"
+
+sceptre_user_data:
+  Distro: "ubuntu"
+  OpenPorts: ["80", "443", "22"]
+parameters:
+  Department: "CompOnc"
+  Project: "htan"
+  OwnerEmail: "bruno.grande@sagebase.org"
+  InstanceType: "i3en.2xlarge"
+  AMIId: "ami-0b7906ab614596e7e"  # https://github.com/Sage-Bionetworks-IT/packer-base-ubuntu-bionic/releases/tag/v1.0.9
+  VolumeSize: "1000"
+  KeyName: "EmoryHealthcare-dgutman"  # Imported via AWS Management Console
+  VpcName: "htan-vpc"
+  VpcSubnet: "PublicSubnet"
+  JcConnectKey: !ssm "/infra/JcConnectKey"
```

Comment:
This should not be a jumpcloud connected host. AFAIK dgutman will have an SSH key pair outside of JC. In prod, we would not deploy a JC-connected web server not use the web server as a jumpbox.

Reply:
I've gotten rid of these params.